api/src/main/java/org/openmrs/BaseCustomizableData.java

```diff
@@ -83,7 +83,6 @@ public void addAttribute(A attribute) {
 		if (getAttributes() == null) {
 			setAttributes(new LinkedHashSet<A>());
 		}
-		// TODO validate
```

Comment:
I had to squash everything, including a merge of upstream master, which gave those TODO removals.